<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryReportDialog</class>
 <widget class="QDialog" name="EntryReportDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entry Report</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="fromDate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="toDate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>EntryReportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>822</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>852</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>EntryReportDialog</receiver>
   <slot>fromDateChanged(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>EntryReportDialog</receiver>
   <slot>toDateChanged(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton</sender>
   <signal>clicked()</signal>
   <receiver>EntryReportDialog</receiver>
   <slot>onExportButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fromDateChanged(QDate)</slot>
  <slot>toDateChanged(QDate)</slot>
  <slot>onExportButtonClicked()</slot>
 </slots>
</ui>
